burst-gradle-plugin/src/test/kotlin/app/cash/burst/gradle/TestInterceptorGradlePluginTest.kt

```diff
@@ -222,7 +222,7 @@ class TestInterceptorGradlePluginTest {
     with(tester.readTestSuite("app.cash.burst.tests.InterceptorAndBurstConstructorTest_false")) {
       assertThat(systemOut).isEqualTo(
         """
-        |intercepting false test
+        |intercepting false app.cash.burst.tests.InterceptorAndBurstConstructorTest_false.test
```

Comment:
This one is the most interesting. Burst’s two features intersect here, and we emit the parameterized test class name